### ISSUE TITLE
Pantry: Sameas searchable drop down box

### ISSUE THREAD

**Issue by 1fastgranada** [wdross/sites#2]
Add a drop down box to choose sameas from. Make the box search as you type letters.

**Comment by wdross** [comment]
Should the search be on UPC, description or either?

**Comment by 1fastgranada** [comment]
Yes, good idea. I vote both, but probably also include "brand" as well. I'd like to be able to search for "cheerios" or "rice" or "Oreo"
